Assets/Scripts/Augment/GunFactory.cs

```diff
@@ -66,6 +65,13 @@ public static GameObject InstantiateGun(Item bodyPrefab, Item barrelPrefab, Item
         return gun;
     }
 
+    public static void DesubscribeAnimators(GameObject gun)
+    {
+        var firstPersonGunController = gun.GetComponent<GunFactory>().GunController;
+        firstPersonGunController.onFireStart = null;
+        firstPersonGunController.onReload = null;
```

Comment:
also... couldn't this just as well call the ginormous unsub method inside guncontroller?

Reply:
No. This *has* to be done in this scope, unless we refactor the guncontroller to subscribe the delegate in the same way augmentanimators do

Reply:
Yeah I see :))))))
Seems fine to save that for a future PR (hopefully soon™️)